### PR TITLE
Add Suppress from Discovery field to PCI View/Edit

### PR DESCRIPTION
**Pull request by aditya-matukumalli**

### CODE REVIEW

File: src/components/EResourceSections/PCIFormInfo.js [hunk 1]
```diff
@@ -49,6 +50,25 @@ export default class PCIFormInfo extends React.Component {
             />
           </Col>
         </Row>
+        <Row>
+          <Col xs={12}>
+            <Field
```
Comment (doytch) [inline src/components/EResourceSections/PCIFormInfo.js:55]: If this was c/ped from my code, [it doesn't need to be this complicated](https://github.com/folio-org/ui-agreements/pull/537).

Comment (aditya-matukumalli) [reply src/components/EResourceSections/PCIFormInfo.js:53]: Yeah, fixed it.

File: src/components/EResourceSections/PCIInfo.js [hunk 2]
```diff
@@ -87,6 +87,15 @@ export default class PCIInfo extends React.Component {
             </KeyValue>
           </Col>
         </Row>
+        <Row>
```
Comment (doytch) [inline src/components/EResourceSections/PCIInfo.js:90]: Can you have this mirror the four-column layout in the edit page?

Comment (aditya-matukumalli) [reply src/components/EResourceSections/PCIInfo.js]: Done

**Pull request merged**